app/assets/javascripts/admin/addon/components/modal/start-backup.js

```diff
@@ -1,16 +1,37 @@
 import Component from "@glimmer/component";
 import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+import I18n from "I18n";
 
 export default class StartBackup extends Component {
-  @action
-  startBackupWithUploads() {
-    this.args.model.startBackup(true);
-    this.args.closeModal();
+  @service siteSettings;
+  @tracked includeUploads = true;
+
+  get canManageUploadsInBackup() {
+    return (
+      !this.siteSettings.enable_s3_uploads ||
+      this.siteSettings.include_s3_uploads_in_backups
+    );
+  }
+
+  get warningCssClasses() {
+    return "";
+  }
+
+  get warningMessage() {
+    if (
+      this.siteSettings.enable_s3_uploads &&
+      !this.siteSettings.include_s3_uploads_in_backups
+    ) {
+      return I18n.t("admin.backups.operations.backup.upload_warning");
```

Comment:
non-blocking: since the logic is scoped to s3 only, I wonder if it would be more precise to have this key be `s3_upload_warning` ?